### PR TITLE
upstream merge 7/12

### PR DESCRIPTION
**Pull request by deltanedas**
## Notable changes
- succumb action has 10 second delay to avoid poison cheesing
- there is an admin remarks button in the lobby now
- singulo generator wont work without containment fields
- exped fixes, i never cherry picked them early so they were still broken :trollface:
- dim light bulbs, they can probably replace the deltav maints lights
- derelict borg event, borg with funny laws
- wizard access for admemes
- station anchor beacon for mapping
- greytide virus, the announcement is renamed

## Notable exclusions
- salv vendor grappling hook increase since grappling hooks are commented out from the vendor
- burnt tile removal because tile migrations dont work

**Changelog**
:cl:
- add: Merged mid november-december changes from upstream.

### CODE REVIEW

**Comment by Avalon-Proto** [comment]
Greytide virus???

**Comment by deltanedas** [comment]
bolts random doors open, probably bad so disabled it

**Comment by therealDLondon** [comment]
Greytide virus is a /TG/ event and it fucking SUCKS but an AI can fix it easy if they're on the ball and people call out bolted doors

**Comment by Avalon-Proto** [comment]
Oh weird. I thought it was gonna be like some thing that turns people bald and sticks a gas mask on their face lol

**Comment by Lyndomen** [comment]
Direction reviewing, will have feedback in a day on greytide. Im also dubious on this

> singulo generator wont work without containment fields

so im getting some feedback. thoughts?

**Comment by MilonPL** [comment]
> so im getting some feedback. thoughts?

supposed to make singuloosing as an antag roundstart harder, you can emag it to bypass it

**Comment by Radezolid** [comment]
Wasn't it to prevent non antags from singuloosing shift start?

**Comment by MilonPL** [comment]
yes that's what i meant i woke up 10 minutes ago (since antags can just emag it)

**Review by Lyndomen — DISMISSED** [review]
> singulo generator wont work without containment field
We are split down the middle on this one, we don't mind it but also don't really care

Greytide virus is fine, but please relabel to something like
"Rogue subroutine detected in door servos. Check all door bolts for malfunction"

**Comment by deltanedas** [comment]
bruh i squashed again

File: Content.Client/UserInterface/Systems/MenuBar/Widgets/GameTopMenuBar.xaml [hunk 1]
```diff
@@ -11,25 +11,29 @@
            Orientation="Horizontal"
            HorizontalAlignment="Stretch"
            VerticalAlignment="Top"
-           SeparationOverride="5"
 >
+    <GridContainer
+        Rows="2"
+        HSeparationOverride="5"
+        VSeparationOverride="5"
+        HorizontalExpand="True">
```
Comment (apocist) [inline Content.Client/UserInterface/Systems/MenuBar/Widgets/GameTopMenuBar.xaml:19]: Is the bulging buttons taking up the chat window intentional? 
image

Comment (deltanedas) [reply Content.Client/UserInterface/Systems/MenuBar/Widgets/GameTopMenuBar.xaml:19]: thats how it used to be just less wide, can be reverted

**Pull request merged**